content/docs/pipelines/sdk/python-based-visualizations.md

```diff
@@ -62,7 +62,7 @@ arguments.
 
 ### Predefined TFX visualization
 
-<video width="960" height="540" controls>
+<video width="100%" max-width="100%" height="auto" max-height="100%" controls>
```

Comment:
These settings fit the screen cast on the page (laptop screen). I have not verified a phone sized screen.
There are most likely other ways of making this element responsive.